ops/internal/deployer/deployer.go

```diff
@@ -45,8 +45,8 @@ func (d *OpDeployer) BuildBinary() (string, error) {
 	// 1. Remove tag:// prefix if present
 	// 2. Remove any -rc.X suffix for version matching
 	contractsKey := strings.TrimPrefix(d.l1ContractsRelease, "tag://")
-	rcSuffixRegex := regexp.MustCompile(`-rc\.[0-9]+$`)
-	contractsKey = rcSuffixRegex.ReplaceAllString(contractsKey, "")
+	// rcSuffixRegex := regexp.MustCompile(`-rc\.[0-9]+$`)
+	// contractsKey = rcSuffixRegex.ReplaceAllString(contractsKey, "")
```

Comment:
This is a hack I needed because some versions of op-deployer actually check for a tag which has the rc suffix.